pkg/nuctl/command/common/helpers.go

```diff
@@ -16,7 +16,7 @@ import (
 func FormatFunctionIngresses(function platform.Function) string {
 	var formattedIngresses string
 
-	ingresses := function.GetIngresses()
+	ingresses, _ := function.GetIngresses()
```

Comment:
As I wrote below, I think we should return errors on GetIngresses() because there are type casts that may panic
It should never happen of course, but I think that except validating before, it is good to return error on GetIngresses() in case for some reason we missed a flow - instead of just crashing